src/core/model.js

```diff
@@ -532,7 +534,11 @@ count: ${this.stats.profileFrameCount}`
       const attributeTable = this._getAttributesTable({
         header: `${this.id} attributes`,
         program: this.program,
-        attributes: Object.assign({}, this.geometry.attributes, this.attributes)
+        attributes: Object.assign(
+          {},
+          this.geometry ? this.geometry.attributes : {},
```

Comment:
`this.geometry && this.geometrt.attributes` would work. `Object.assign` handles empty arguments.